lib/membrane_rtc_engine/tees/tee.ex

```diff
@@ -175,17 +178,16 @@ defmodule Membrane.RTC.Engine.Tee do
         _ctx,
         state
       ) do
-    %{current_variant: variant} = state.routes[pad]
+    %{current_variant: current_variant} = state.routes[pad]
 
-    if Map.has_key?(state.inactive_variants, variant) do
-      Membrane.Logger.debug("""
-      Endpoint #{endpoint_id} requested keyframe for inactive track variant: #{variant}. \
-      Track #{inspect(state.track)}. Ignoring.
+    if current_variant do
+      Membrane.Logger.warn("""
+      Endpoint #{endpoint_id} requested keyframe but we don't send any variant to it. Ignoring.
       """)
 
-      {:ok, state}
+      {{:ok, event: {Pad.ref(:input, {state.track.id, current_variant}), event}}, state}
```

Comment:
Are you sure about this if? If `current_variant` is not nil, we are logging that we send it any variant and we are ignoring it, so we send the event on the pad.

Reply:
Yeah, that's obviously a mistake